web/auth/auth_test.go

```diff
@@ -31,18 +33,18 @@ func TestOAuth2Login(t *testing.T) {
 		Expect(t).
 		Status(http.StatusUnauthorized).
 		End()
-	// No existing session cookie.
+	// No existing auth cookie.
 	apitest.New().Debug().
 		HandlerFunc(auth.OAuth2Login(logger, authConfig, "")).
 		Get(web.Auth).
 		Expect(t).
 		Status(http.StatusTemporaryRedirect).
 		End()
-	// Outdated session with expected name should not break API.
+	// Outdated auth cookie with expected name should not break API.
 	apitest.New().Debug().
 		HandlerFunc(auth.OAuth2Login(logger, authConfig, "")).
 		Get(web.Auth).
-		Cookie("session", "empty").
+		Cookie("auth", "empty").
```

Comment:
We should ideally use a global/exported `const` instead of `"auth"`. There are two:
- `auth.CookieName`
- `tokens.authCookieName`

Suggest to rename the latter and use it as `tokens.CookieName` or `tokens.AuthCookieName`. This is because `auth` cannot be imported into `tokens` package.